File: Objetivo 1/Objetivo1_Exercicios_PHP/control/AppConta.php
<?php

/*
 * Observe que em PHP não é necessário ter um comportamento main em uma classe.
 */

include "../model/Conta.php";

$conta1 = new Conta();
echo "\n$conta1"; //observe que o construtor padrão inicializa todos os atributos como uma string vazia (a linguagem não é tipada)
$conta1->deposita(1000.00);
echo "\n$conta1";

$conta2 = new Conta();
$conta2->deposita(2000.00);
echo "\n$conta2"; //observe que o construtor padrão inicializa todos os atributos como uma string vazia (a linguagem não é tipada)
$conta2->saca(200.00);
echo "\n$conta2";

File: Objetivo 1/Objetivo1_Exercicios_Java/src/control/AppConta.java
package control;

import model.Conta;

/**
 *
 * @author vagner
 */
public class AppConta {
    public static void main(String[] args) {
        Conta conta1 = new Conta();
        System.out.println(conta1);
        conta1.deposita(5000.00);
        System.out.println(conta1);
        
        
        Conta conta2 = new Conta(1000.00);
        System.out.println(conta2);
        conta2.deposita(5000.00);
        System.out.println(conta2);
    }
   
}
